lib/utils/result_formatter.ts

```diff
@@ -5,6 +5,17 @@ import { CoordinateUtils } from "./coordinate_utils";
  * Class to format the results of common fields
  */
 export class ResultFormatter {
+    static wheelsUp(decodeResult: DecodeResult, time: number) {
+      decodeResult.raw.wheels_off_time = time;
+      const date = new Date(time * 1000);
+      decodeResult.formatted.items.push({
+        type: 'time_of_day',
+        code: 'WHEELS_OFF',
```

Comment:
What is Wheels Off? Is that a general guess? Or do you think this fits into OOOI (i.e. OUT OFF ON IN)? If so, then we should standardize there, and I have some other terminology for related based on flight events.

Reply:
It is! These positions line up with the middle of the runway and match adsb tracks

Reply:
And yes, I'm assuming this is the OFF of the OOOI phases you're referring to